core/payment/src/payment_sync.rs

```diff
@@ -25,13 +29,26 @@ use ya_service_bus::{timeout::IntoTimeoutFuture, typed, Error, RpcEndpoint};
 
 use crate::dao::{DebitNoteDao, InvoiceDao, InvoiceEventDao, PaymentDao, SyncNotifsDao};
 
-const SYNC_NOTIF_DELAY_0: Duration = Duration::from_secs(30);
-const SYNC_NOTIF_RATIO: u32 = 6;
-const SYNC_NOTIF_MAX_RETRIES: u32 = 7;
 const REMOTE_CALL_TIMEOUT: Duration = Duration::from_secs(30);
 
+fn remove_allocation_ids_from_payment(payment: &Payment) -> Payment {
+    // We remove allocation ID from syncs because allocations are not transferred to peers and
+    // their IDs would be unknown to the recipient.
+    let mut payment = payment.clone();
+    for agreement_payment in &mut payment.agreement_payments.iter_mut() {
+        agreement_payment.allocation_id = None;
+    }
+
+    for activity_payment in &mut payment.activity_payments.iter_mut() {
+        activity_payment.allocation_id = None;
+    }
+
+    payment
+}
+
 async fn payment_sync(
     db: &DbExecutor,
+    current_node_id: NodeId,
```

Comment:
I would call it current. Not sure what it means in this context. Maybe just `node_id` or owner_id`